test/LAMMPS_jll/.pkg/select_artifacts.jl

```diff
@@ -0,0 +1,27 @@
+# Note: This file is executed from `Pkg` in an isolated environment.
+#       We add the current package and stdlib to the load path so that
+#       `platform_augmentation.jl` can use the dependencies of the project,
+#       and `select_artifacts.jl` needs TOML, and Artifacts.
+# Warn: Since this is executed in a sandbox, the current preferences are not
+#       propagated and thus no decision can be made based on them.
+#       Use LazyArtifacts in this case.
+
+push!(Base.LOAD_PATH, "@stdlib")
+push!(Base.LOAD_PATH, dirname(@__DIR__))
```

Comment:
As Elliot and I just discovered this is not sufficient since we might be in a partially instantiated case.